.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Environment (please provide info about your environment):**
+ - Installation type: [e.g. Linux-Flatpak/Linux-Tarball/Linux-AppImage/Windows/WSL/Mac/Mac or Container-Image/In-Cluster/Helm]
+ - Headlamp Version: [X.Y.Z]
+ - Other: [e.g. any other relevant info] [REMOVE IF NON-APPLICABLE] 
+
+**Additional context**
+[REMOVE IF NON-APPLICABLE] Add any other context about the problem here.
+
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+[REMOVE IF NON-APPLICABLE] If applicable, add screenshots to help explain your problem.
```

Comment:
One feature I liked in the old issue template was that all the text to be removed was encased in `[ ]`.

- `[REMOVE IF NON-APPLICABLE] If applicable, add screenshots to help explain your problem.`
- `[REMOVE IF NON-APPLICABLE If applicable, add screenshots to help explain your problem.]`
- `[Add screenshots to help explain your problem.]`

IMHO It's a little bit easier to visually see what to delete if there's a marker at the end and start.

It's also easier if there is consistency throughout the whole thing. So that all text that needs to be removed is wrapped in `[]`.

Reply:
Some are wrapped in [] now. It would be good if all parts are wrapped in [].